### PR TITLE
[GLS Bank] Rewrite, activate ruleset

### PR DESCRIPTION
**Pull request by rugk**

### CODE REVIEW

**Comment by rugk** [comment]
Why does the CI test fail? Respectively why only [the one](https://travis-ci.org/EFForg/https-everywhere/jobs/110482855) which uses Firefox ESR?

**Comment by rugk** [comment]
Any news?

**Comment by rugk** [comment]
The CI test should not fail. Please check my PR manually.

**Comment by J0WI** [comment]
Please also check:
laden.gls.de
sharedichdrum.gls.de
mikrokreditfonds.gls.de
news.gls.de

**Comment by rugk** [comment]
Done

File: src/chrome/content/rules/GLS.de.xml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 <!--
-
+    Nonfunctional subdomains:
+        - laden            -> wrong domain
+        - sharedichdrum    -> too long
```
Comment (J0WI) [inline src/chrome/content/rules/GLS.de.xml]: Please use 'no secure protocols supported' to differ from time outs.

Comment (rugk) [inline src/chrome/content/rules/GLS.de.xml]: Ah, that does the "SSL_ERROR_RX_RECORD_TOO_LONG" actually mean? A cipher/tls version mismatch?

Comment (J0WI) [inline src/chrome/content/rules/GLS.de.xml]: The server sends plain answer (no tls) over port 443 which results in an error.

**Pull request merged**